RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,476 @@
+# React Native Paper Component Templates
+
+This Document contains a set of style override in components around various [React Native Paper](https://callstack.github.io/react-native-paper/index.html) components.
+
+By default, the theming mechanism provided by RNP is very minimal and does not allow us to style components precisely the way we want for Brightlayer UI applications. To circumvent this issue, we create a style overriden components that you can use bring them in alignment with Brightlayer UI applications
+
+These style overriden components simplify this process so that the end user do not create the component designs to bring them in alignment with Brightlayer UI applications.
```

Comment:
Ensure that you refer our readers consistently. On line 5 you called then "you", while on line 7 you called the same group of people "the end user".

Reply:
There are also some grammatical errors in these three opening paragraphs. You may want to plug them into a grammar checker